releasenotes/notes/add_chocolatey_pkg_generator-9c1ba11cf3a80677.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    Add chocolatey package generation in pipeline.
```

Comment:
Changes to the pipeline aren't relevant to users of the Agent. Something like "The Agent is now available on Chocolatey for Windows" would be more useful for an end user, but I don't think we announce these things via our changelog normally. At least we didn't do it for ARM for example. Wdyt @olivielpeau?

Reply:
I agree that the release note should mention things that are available to end users, not internal changes. (so writing `The Agent is now available on Chocolatey for Windows` is much better 👍 )

If the chocolatey package will indeed be made publicly available in the next agent release, I think it's worth mentioning in release notes as well (even if it should also be announced via other channels of course :) ). A link to the announcement and/or some docs and/or the chocolatey page would also make sense in the release note IMO.